mirrulations-work-server/src/mirrserver/work_server.py

```diff
@@ -150,13 +150,16 @@ def get_job(workserver):
     job = json.loads(workserver.redis.lpop('jobs_waiting_queue'))
     job_id = job['job_id']
     url = job['url']
+    agency = job['agency'] if job.get('agency') else "other_agency"
+    reg_id = job['reg_id'] if job.get('reg_id') else "other_reg_id"
```

Comment:
Too many local variables in this function because there is 1 of each for each field in the json, this is the only testing issue so far.